### PR TITLE
Configure Spring in Movie-Shop project

### PR DESCRIPTION
**Pull request by IrynaDim**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,11 +35,63 @@
...
+        <dependency>
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
+            <version>1.2.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.10.2</version>
+        </dependency>
```
Comment (andriylagun) [inline pom.xml]: Do we need that dependency at now?

**Review by andriylagun — APPROVED** [review]
Good job!

File: pom.xml [hunk 2]
```diff
@@ -62,28 +110,30 @@
...
-                        <encoding>${project.build.sourceEncoding}</encoding>
-                    </configuration>
-                </plugin>
             </plugins>
         </pluginManagement>
     </build>
 </project>
+
+
+
```
Comment (RomanDubovskyi) [inline pom.xml]: do we need 4 empty lines here

File: pom.xml [hunk 3]
```diff
@@ -34,11 +35,58 @@
...
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp</groupId>
+            <artifactId>javax.servlet.jsp-api</artifactId>
```
Comment (RomanDubovskyi) [inline pom.xml]: do we have we need these dependencies, if we have no jsp?

File: src/main/java/com/dev/cinema/config/WebConfig.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"com.dev.cinema.controllers"})
+public class WebConfig {
+
+}
+
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/config/WebConfig.java]: redundant line

File: pom.xml [hunk 5]
```diff
@@ -34,11 +35,52 @@
...
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet.jsp.jstl</groupId>
+            <artifactId>javax.servlet.jsp.jstl-api</artifactId>
```
Comment (RomanDubovskyi) [inline pom.xml]: what about this one, do we need it?

File: src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java [hunk 6]
```diff
@@ -1,17 +1,19 @@
...
-import com.dev.cinema.lib.Service;
 import com.dev.cinema.model.Movie;
 import com.dev.cinema.service.MovieService;
 import java.util.List;
+import org.springframework.stereotype.Service;
 
 @Service
 public class MovieServiceImpl implements MovieService {
-    @Inject
     private MovieDao movieDao;
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java]: put final here as well to keep code consistent and check other classes for this keyword

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**